wikimedia/wikipedia-ios
# Article descriptions - Add casing warning label for non-EN wikis

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T261977

### Notes
This adds an extra warning label for users that begin their non-EN wiki article descriptions with a lowercase letter. It does not block publishing though, in case they are starting with a proper noun. Do not review until #3880 is merged and this PR is updated with `main` to clean up the diff.

### Test Steps
1. On a non-EN wiki article, edit or add an article description. Confirm that if description starts with an uppercase letter, you see the red label about casing as stated in the Phab task. Confirm if you enter > 90 characters, you see both the casing labels and the length labels. Confirm if you enter > 90 characters and the description begins with a lowercase letter, you only see the length label.
2. On an EN wiki article, edit or add an article description. Confirm you only see a length label if you enter > 90 characters, and you never see the red casing label.

### Screenshots/Videos *(if applicable)*
(Non-EN examples)
![Screen Shot 2021-03-10 at 1 38 08 PM](https://user-images.githubusercontent.com/3620196/110687325-fed6a980-81a5-11eb-946b-e050cb370a4c.png)
![Screen Shot 2021-03-10 at 1 38 03 PM](https://user-images.githubusercontent.com/3620196/110687311-fb432280-81a5-11eb-81fc-8a5e9d8d4fd0.png)
![Screen Shot 2021-03-10 at 1 37 59 PM](https://user-images.githubusercontent.com/3620196/110687338-01390380-81a6-11eb-9e80-ec2452e29743.png)
